cmd/restart.go

```diff
@@ -21,6 +21,7 @@ func init() {
 
 	skipBuild = restartCmd.Flags().BoolP("skip-build", "s", false, "Skip the build phase")
 	noWatch = restartCmd.Flags().Bool("no-watch", false, "Disable autorestart")
+	noPrompt = restartCmd.Flags().BoolP("no_prompt", "n", false, "Skip confirmation prompt")
```

Comment:
The mix of snake case and hyphens seems a little off when they're put together.

This might make more sense as --force or -f, and then we can deprecate the other use(s) of no_prompt to replace with force.